src/postprocessing/least_squares.jl

```diff
@@ -226,27 +244,45 @@ function newtonls(res, w, x0, niters=5)
     npar = length(x0)
     # Mean square residual
     Q = chi2(res, w)/nobs
+    # Vector of x 
+    x = Matrix{T}(undef, niters + 1, npar) 
```

Comment:
Same comments here for exploiting "column major" design in Julia, and using views...

Reply:
Same as above.